torch/csrc/jit/passes/quantization_patterns.h

```diff
@@ -0,0 +1,77 @@
+#pragma once
+
```

Comment:
Nit: please add
```
#include <string>
#include <unordered_map>
```
Otherwise if the include order in `quantization.cpp` changes for some reason it might fail to build. Also, might be worth considering making it in a cpp file (vs in a .h).

Reply:
I'll move to cpp in a separate PR since this one is in the bottom of a stack